crates/solidity/testing/snapshots/cst_output/ContractMembersList/incomplete_recovery/generated/0.8.0-failure.yml

```diff
@@ -0,0 +1,106 @@
+              - OpenParen (Token): "(" # 10..11
+              - ParametersList (Rule): # 11..24 "uint arg, int"
+                  - Parameter (Rule): # 11..19 "uint arg"
+                      - TypeName (Rule): # 11..15 "uint"
+                          - UnsignedIntegerType (Token): "uint" # 11..15
+                      - Identifier (Token): "arg" # 16..19
+                  - Comma (Token): "," # 19..20
+                  - Parameter (Rule): # 20..24 " int"
+                      - TypeName (Rule): # 20..24 " int"
+                          - SignedIntegerType (Token): "int" # 21..24
+              - Error: ", true" # 24..30
+              - CloseParen (Token): ")" # 30..31
+          - Block (Rule): # 31..94 " {\n  uint val;\n  break;\n  unchecked { /* hello */ ..."
+              - OpenBrace (Token): "{" # 32..33
+              - StatementsList (Rule): # 34..92 "  uint val;\n  break;\n  unchecked { /* hello */ }\n ..."
+                  - Statement (Rule): # 34..46 "  uint val;\n"
+                      - ExpressionStatement (Rule): # 34..46 "  uint val;\n"
+                          - Expression (Rule): # 34..40 "  uint"
+                              - UnsignedIntegerType (Token): "uint" # 36..40
+                          - Error: "val" # 41..44
+                          - Semicolon (Token): ";" # 44..45
+                  - Statement (Rule): # 46..55 "  break;\n"
+                      - BreakStatement (Rule): # 46..55 "  break;\n"
+                          - BreakKeyword (Token): "break" # 48..53
+                          - Semicolon (Token): ";" # 53..54
+                  - Statement (Rule): # 55..83 "  unchecked { /* hello */ }\n"
+                      - UncheckedBlock (Rule): # 55..83 "  unchecked { /* hello */ }\n"
+                          - UncheckedKeyword (Token): "unchecked" # 57..66
+                          - Block (Rule): # 66..83 " { /* hello */ }\n"
+                              - OpenBrace (Token): "{" # 67..68
+                              - LeadingTrivia (Rule): # 68..81 " /* hello */ "
+                                  - MultilineComment (Trivia): "/* hello */" # 69..80
+                              - CloseBrace (Token): "}" # 81..82
+                  - Statement (Rule): # 83..92 "  break;\n"
+                      - BreakStatement (Rule): # 83..92 "  break;\n"
+                          - BreakKeyword (Token): "break" # 85..90
+                          - Semicolon (Token): ";" # 90..91
+              - CloseBrace (Token): "}" # 92..93
+      - StateVariableDefinition (Rule): # 94..154 "\nfunction nobody() public\n\nfunction noargs public ..."
+          - TypeName (Rule): # 94..103 "\nfunction"
+              - FunctionType (Rule): # 94..103 "\nfunction"
+                  - FunctionKeyword (Token): "function" # 95..103
+                  - Error: "" # 103..103
+          - Error: "nobody() public\n\nfunction noargs public {\n\tbreak" # 104..152
+          - Semicolon (Token): ";" # 152..153
```

Comment:
We can't have this on main if this doesn't produce a complete CST covering every char of the input.